src/datasets/providers/env-var.ts

```diff
@@ -0,0 +1,27 @@
+import EnvVarSvg from '@site/static/img/env-var-no-fill.svg';
```

Comment:
This looks very similar to the `user-defaults-no-fill.svg`. Perhaps that could be renamed something generic like `gear-icon.svg` and used in both places.

Reply:
You're right, the logo was renamed gear-icon.svg and is now used in both places.